### DISCUSSION TITLE
Change the Icons and Everything Else modified back to normal

### DISCUSSION THREAD

**Discussion by DatGamerYoloYT** [DEMOLITIONDON96/Demolition-Engine#29] [Ideas (Other)]
title says it all.

I just don't like the new stuff and i'm basically changing it all back but if there could either be a option to toggle the normal ui or everything getting reverted back that would be nice

**Comment by ThatOneSillyGuyDon** [comment]
I’ll see what we can do

**Reply by DatGamerYoloYT** [reply]
:D
